### DISCUSSION TITLE
Is there a method to list all resources in AWS?

### DISCUSSION THREAD

**Discussion by golden-rabbit-123579** [community/community#135570] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I’m looking to gather information for all regions and all types of resources, such as EC2 instances, VPCs, APIs in API Gateway, etc. I want to compile a complete list of resources for my account, as it's challenging for me to identify which resources I can release at this time.

**Comment by everforyou** [comment, marked as answer]
Yes, you can use the Tag Editor.

Set "Regions" to "All Regions" and "Resource Types" to "All supported resource types," then click on "Search Resources."

**Reply by golden-rabbit-123579** [reply]
I will try like that, @BrotherRabbit91

**Comment by golden-rabbit-123579** [comment]
It works. thanks for your support, BrotherRabbit91
